actors/builtin/miner/miner_actor.go

```diff
@@ -460,17 +409,19 @@ func (a Actor) ExtendSectorExpiration(rt Runtime, params *ExtendSectorExpiration
 }
 
 type TerminateSectorsParams struct {
-	sectorNumbers []abi.SectorNumber
+	Sectors *abi.BitField
 }
 
 func (a Actor) TerminateSectors(rt Runtime, params *TerminateSectorsParams) *adt.EmptyValue {
 	var st State
 	rt.State().Readonly(&st)
 	rt.ValidateImmediateCallerIs(st.Info.Worker)
 
+	sectorNos := bitfieldToSectorNos(rt, params.Sectors)
```

Comment:
We should set a max count here, probably independent of any bitfield limitatinos.

Reply:
Yeah, its set in the bitfieldToSectorNos method. Do you mean other than that one?